quickwit/quickwit-serve/Cargo.toml

```diff
@@ -32,6 +32,7 @@ serde_qs = { workspace = true }
 serde_with = { workspace =  true }
 termcolor = { workspace = true }
 thiserror = { workspace = true }
+tantivy = { workspace = true }
```

Comment:
same question, can we avoid this?

Reply:
we need to convert the value to tantivy::Type